crates/sdk/src/network-v2/client.rs

```diff
@@ -95,6 +97,76 @@ impl NetworkClient {
         Ok(res.into_inner().nonce)
     }
 
+    /// Get the verifying key hash from a verifying key. The verifying key hash is used to identify
+    /// a program.
+    pub fn get_vk_hash(vk: &SP1VerifyingKey) -> Result<Vec<u8>> {
```

Comment:
Wondering if we should use SP1VerifyingKeyHash([u32; 8]) or something

Reply:
will keep this in mind